frontend-ssr/components/cli_git/match_commands.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+class MatchWords extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            value: '',
+            message: '',
+        }
+
+        this.handleInputChange = this.handleInputChange.bind(this);
+        this.handleMatch = this.handleMatch.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+    handleSubmit(event) {
+        event.preventDefault();
+    }
+    handleInputChange(event) {
+        this.setState({
+            value: event.target.value
+        });
+    }
+
+    handleMatch() {
+        let inputValue = this.state.value;
```

Comment:
Use destructuring pattern. 

`const {a, b, c} = this.props` 

instead of

```
const a  = this.props.a;
const b = this.props.b;
const c = this.props.c;
```